samples/dpdk_app/sriov/sriov-pod-1.yaml

```diff
@@ -48,6 +50,11 @@ spec:
         - path: "annotations"
           fieldRef:
             fieldPath: metadata.annotations
+        - path: "hugepageSize"
+          resourceFieldRef:
+            containerName: dpdk-app
+            resource: requests.memory
```

Comment:
I just noticed that, with `requests.memory` above, dpdk-app is using normal memory instead of hugepage inside container, is that correct?

Reply:
No, `requests.memory` is how the hugepage size is requested, so that is what is passed through:
```
    resources:
      requests:
        memory: 1Gi
        #memory: 2Mi
```

Reply:
No, requests.memory is how the hugepage size is requested, so that is what is passed through:
```
    resources:
      requests:
        memory: 1Gi
        #memory: 2Mi
```

Reply:
Misunderstanding on my part. I have been adding hugepages incorrectly all this time. There is a patch underway upstream to add the hugepage fields to the downward API. Once those are in, a new patch will be submitted. Abandoning this PR now.